nls/kappnav.properties

```diff
@@ -226,6 +226,14 @@ action.url.deployment.kabcolldoc.text= View Collection Doc
 action.url.deployment.kabcolldoc.desc= View Kabanero Collection Documentation.
 action.url.deployment.kabcollcode.text= View Collection Code
 action.url.deployment.kabcollcode.desc= View Kabanero collection source code.
+action.url.nodejs.deployment.appmetric.text= View AppMetric-Dash
```

Comment:
appmetric-dash is the name of the Node package but since this is user text in our UI I think it should be "Node Application Metrics Dashboard"

Reply:
Following teh surroundding examples, the desc message could add "for Deployment" or "for DeploymentConfig", e.g., "View Node Application Metrics Dashboard for Deployment"

Reply:
Updated